Several updates in preparation for the 4.8.0 release. (#320)
AxonIQ/reference-guide: axon-framework/deadlines/event-schedulers.md

Just as the `QuartzEventScheduler`, the `AxonServerEventScheduler` is a reliable and enterprise-worthy implementation of the `EventScheduler` interface.
Creating a `AxonServerEventScheduler` can be done through its builder, whose sole requirement is the `AxonServerConnectionManager`.

> The `DbScheduler` is a reliable and enterprise-worthy implementation.

Reviewer: I think it might be nicer to start this sentence differently from the JobRunr description.
Perhaps something like this:

```suggestion
The `DbSchedulerEventScheduler` is similar in convenience compared to the JobRunr- and the Axon Server `EventScheduler` implementations.
```

It offers several ways to persist the scheduled jobs, and has good integration with Spring Boot.
It's pretty simple, at it only needs one table. For most sql databases the sql to set the table is [available](https://github.com/kagkarlsson/db-scheduler/tree/master/db-scheduler/src/test/resources/com/github/kagkarlsson/scheduler)

> Compared to JobRunr, it let you set a poll interval for new tasks more freely.

Reviewer: ```suggestion
Compared to JobRunr, it lets you set a poll interval for new tasks more freely.
```

> another advantage compared to JobRunr is that adding a task will become part of the current transaction, if a `TransactionManager` is set.

Reviewer: ```suggestion
Another advantage compared to JobRunr is that adding a task will become part of the current transaction if a `TransactionManager` is set.
```

The biggest things missing compared to JobRunr are a dashboard, and support for multiple NoSQL databases.

> It needs to be configured with a db-scheduler `Scheduler`, an `EventBus` and a `Serializer`.
> The data for an event can be serialized in either a binary format, or a human-readable format. This is done via the `useBinaryPojo` on the builder.

Reviewer: ```suggestion
The `DbSchedulerEventScheduler` needs to be configured with a db-scheduler `Scheduler`, an `EventBus`, and a `Serializer`.
The data for an event can be serialized in either a binary format or in a human-readable format. Switching between these formats is done via the `useBinaryPojo` property on the scheduler's builder.
```

> }
> ```

> For both JobRunr and db-scheduler autoconfiguration is included via the [axon-spring-boot-starter](https://github.com/AxonFramework/AxonFramework/tree/master/spring-boot-starter). This means that if you don't have an `EventScheduler` configured in the application, but you do have a `JobScheduler` bean (in the case of JobRunr), or a `Scheduler` bean (in the case of db-scheduler), an `EventScheduler` will be autoconfigured. In these cases you likely have to set the `axon.axonserver.enabled` property to false, to prevent the creation of the Axon Server `EventScheduler`. 

Reviewer: ```suggestion
For both JobRunr and db-scheduler, auto-configuration is included via the [axon-spring-boot-starter](https://github.com/AxonFramework/AxonFramework/tree/master/spring-boot-starter). 
This means that if you don't have an `EventScheduler` configured in the application, but you do have a `JobScheduler` bean (in the case of JobRunr) or a `Scheduler` bean (in the case of db-scheduler), an `EventScheduler` will be auto-configured for you. 
In these cases, it is recommended to set the `axon.axonserver.enabled` property to `false` to prevent the creation of the `AxonServerEventScheduler`. 
```
